tests/test_metamod.py

```diff
@@ -123,7 +236,7 @@ def test_modflow_dll_present(modflow_dll_devel: Path) -> None:
     assert modflow_dll_devel.is_file()
 
 
-@parametrize_with_cases("metamod_model", cases=".")
+@parametrize_with_cases("metamod_model", cases=".", prefix="case_")
 def test_metamod_develop(
```

Comment:
The name is not very descriptive.

But maybe instead of making the name longer it would be good to add docstrings to the tests?

How about we add docstrings to any non-trivial test?
Maybe longer than 10 lines?